### PR TITLE
Week 19 - Karin Nordkvist (Happy Thoughts Backend/API) ❤️

### PR DESCRIPTION
**Pull request by karinnordkvist**
**Link to my deployed project:** https://happy-thoughts-by-karin.herokuapp.com/ / https://happy-thoughts-by-karin.netlify.app/

**My reflections on how this week's project turned out:**
Great project. Loved how we got to tie the knot for the Happy Thoughts project.

**Things I'd like to have clarified or explained in more detail:**
Nothing, I think. 😊

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,28 @@
...
+* Express
+* MongoDB
+* MongoDB Atlas / Heroku
+
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+- The frondend is live here: https://happy-thoughts-by-karin.netlify.app
+- Backend url: https://happy-thoughts-by-karin.herokuapp.com/
```
Comment (petrame) [inline README.md:28]: I like your readme file, well written and informative ⭐

File: server.js [hunk 2]
```diff
@@ -1,29 +1,90 @@
...
+      .status(404)
+      .json({ message: 'Could not find any thoughts!', errors: err.errors });
+  }
+});
+
+app.post('/thoughts', async (req, res) => {
+  try {
+    const thought = new Thought({
+      message: req.body.message,
+      author: req.body.author,
```
Comment (petrame) [inline server.js:65]: Great! You're taking control over what the user can send in.

File: server.js [hunk 3]
```diff
@@ -1,29 +1,90 @@
...
+    default: 0,
+  },
+  createdAt: {
+    type: Date,
+    default: Date.now,
+  },
+  author: {
+    type: String,
+    default: 'Anonymous',
+  },
```
Comment (petrame) [inline server.js:36]: Nice extra feature.

**Review by petrame — APPROVED** [review]
Great job on the happy thoughts API Karin ⭐
You're endpoints are all working fine as well as you error handling. I've tried to enter incorrect messages both from the website and from the postman and also tried to modify createDate and the number of likes without success.

**Review by KarolCislowski — APPROVED** [review]
This is a very nice and clean code 👍🏻

**Pull request closed without merging**